src/sprite.js

```diff
@@ -190,7 +190,14 @@ class Sprite extends GameObjectClass {
 
     if (this.color) {
       this.context.fillStyle = this.color;
-      this.context.fillRect(0, 0, this.width, this.height);
+      if (this.radius) {
+        this.context.beginPath();
+        this.context.arc(this.radius, this.radius, this.radius, 0, Math.PI * 2);
+        this.context.closePath();
```

Comment:
This can actually be removed as the circle is already closed (since it's a complete circle) and calling `fill` will end the `beingPath` call.